modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/CopyFolderMVCRenderCommand.java

```diff
@@ -31,6 +49,59 @@
 )
 public class CopyFolderMVCRenderCommand extends BaseFolderMVCRenderCommand {
 
+	@Override
+	public String render(
+			RenderRequest renderRequest, RenderResponse renderResponse)
+		throws PortletException {
+
+		try {
+			DLFolder dlFolder = _dlFolderLocalService.getDLFolder(
+				ParamUtil.getLong(renderRequest, "sourceFolderId"));
+
+			long dlFolderSize = _dlFolderLocalService.getDLFolderSize(
+				dlFolder.getTreePath(), dlFolder.getCompanyId(),
+				dlFolder.getGroupId());
+
+			boolean canCopy = _dlSizeLimitConfigurationProvider.canCopy(
```

Comment:
This method can be inlined or is SF change?

Reply:
I'll inline it :)